Towerise/ClientApp/app/components/player/player.component.ts

```diff
@@ -120,8 +116,8 @@ export class PlayerComponent implements AfterViewInit  {
             alert("socket not connected");
         }
         var data = JSON.stringify({
-            x: this.x,
-            y: this.y
+            x: this.playerX,
```

Comment:
I have not tested this, but I imagine that this will not deserialize properly, unless we rename these attributes in the player class on the backend.